deploy/sre-1270-rebalancing-infra-nodes/00-sre-1270-rebalancing-infra-nodes.ServiceAccount.yaml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sre-1270-rebalancing-infra-nodes
+  namespace: openshift-monitoring
```

Comment:
Why we are choosing this namespace? it's nothing related to monitoring.

Reply:
I have discussed with @blrm. We think that the cronjob is used for monitoring the infra workloads, and run the script if it's not balanced. All the other stuff grants permissions to the cronjob. And the workloads the script touches, `alertmanager-main`, `prometheus-k8s`, `prometheus-adapter` are running in `openshift-monitoring`. Only `splunkforwarder-deployment` is in`openshift-security`.  So, it makes sense to put all the stuff in `openshift-monitoring`. How do you think?